dev-python/tablib/tablib-3.0.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Format-agnostic tabular dataset library"
+HOMEPAGE="https://pypi.org/project/tablib/ https://github.com/jazzband/tablib/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="~amd64"
+
+BDEPEND="
+	test? (
+		dev-python/pytest-cov[${PYTHON_USEDEP}]
```

Comment:
What is it required for, then?

Reply:
Yes. Removing this will fails on the way you reported yesterday

Reply:
Ah sorry, I have read wait instead of what 😅

Reply:
Did you sed out the unnecessary pytest args as described in the docs and found in a few dozen other ebuilds?

Reply:
Ok i have seen what you mean, i will fix it asap

Reply:
@mgorny fixed